composer.json

```diff
@@ -12,7 +12,11 @@
     "maknz/slack": "^1.7",
     "sabre/xml": "^1.4",
     "jarednova/mesh": "0.4",
-    "shortlist-digital/agreable-base-theme": "~2.1"
+    "shortlist-digital/agreable-base-theme": "~2.1",
+    "illuminate/queue": "5.2.*",
+    "illuminate/encryption": "5.2.*",
+    "aws/aws-sdk-php": "dev-master",
```

Comment:
Can you pick a tag from here? https://packagist.org/packages/aws/aws-sdk-php how about `3.23.*`?